src/python/grpcio/grpc/_channel.py

```diff
@@ -1104,5 +1100,9 @@ def __del__(self):
         # effect closure of the underlying cygrpc.Channel instance.
         # This prevent the failed-at-initializing object removal from failing.
```

Comment:
1101-1102. This two lines of comment is no longer needed with your update.

Reply:
thanks done